src/report.test.js

```diff
@@ -48,6 +48,9 @@ describe('Report creation', () => {
 
       const allowedMissingFields = [
         'projectId',
+        'disk.totalMiB',
```

Comment:
This test is to guard against a situation where we expect fields in the schema and the agent fails to send them. I'd prefer to only add items to this that are truly optional.

Reply:
They're truly optional until the code is written to provide them in the first place, no?

Reply:
Discussed and agreed that adding to schema + coupled with allowedMissing entry is accepted, with the feature implementation removing it from allowedMissing